# HG changeset patch
# User Gael Guennebaud <g.gael@free.fr>
# Date 1523083519 -7200
# Branch 3.3
# Node ID ba14974d054ae9ae4ba88e5e58012fa6c2729c32
# Parent  8be258ef0b6d09f1c331851c708e5da1429c5c36
Fix cmake scripts with no fortran compiler
(grafted from 607df2a0aa39feecb4334a1a7bd9d07d858598dc)

diff --git a/cmake/language_support.cmake b/cmake/language_support.cmake
--- a/cmake/language_support.cmake
+++ b/cmake/language_support.cmake
@@ -26,7 +26,7 @@
     cmake_minimum_required(VERSION 2.8.0)
     set (CMAKE_Fortran_FLAGS \"${CMAKE_Fortran_FLAGS}\")
     set (CMAKE_EXE_LINKER_FLAGS \"${CMAKE_EXE_LINKER_FLAGS}\")
-    enable_language(${language} OPTIONAL)
+    enable_language(${language})
   ")
   file(REMOVE_RECURSE ${CMAKE_BINARY_DIR}/language_tests/${language})
   file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/language_tests/${language})
